File: pacote-download/Iniciante/Python/uri-1064-PositivosMedia.py
vezesInput = 0
valoresPositivos = 0
soma = 0
for c in range(0,6,1):
    valor = float(input())
    if valor > 0:
        valoresPositivos += 1
        soma += valor
print(f'{valoresPositivos} valores positivos')
print(f'{soma/valoresPositivos:.1f}')

'''lista = list()
vezesInput = 0
while vezesInput < 6:
    valor = float(input())
    if valor > 0:
        lista.append(valor)
    vezesInput += 1
quantidadeNumeros = len(lista)
print(f'{quantidadeNumeros} valores positivos')
print(f'{sum(lista)/quantidadeNumeros:.1f}')
SÓ PARA TREINAR LISTAS
'''

File: pacote-download/Iniciante/JavaScript/Uri1064-PositivosMedia.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n')

var positivos = 0
var soma = 0
for(i=0;i<6;i++){
    valor = Number(lines.shift())
    if(valor > 0){
        positivos += 1
        var soma = soma + valor
    }
}
var media = soma/positivos
console.log(`${positivos} valores positivos`)
console.log(`${media.toFixed(1)}`)